src/main/java/com/bpodgursky/jbool_expressions/And.java

```diff
@@ -29,7 +32,7 @@ private And(Expression<K>[] children, Comparator<Expression> comparator) {
 
   public String toString() {
     if (!cachedStringRepresentation.isPresent()) {
-      cachedStringRepresentation = Optional.of("(" + StringUtils.join(expressions, " & ") + ")");
+      cachedStringRepresentation = Optional.of("(" + String.join(" & ", Arrays.stream(expressions).map(Object::toString).collect(Collectors.toList())) + ")");
```

Comment:
just being fiddly as get familiar with some of the java 8 streaming syntax, but 
````
cachedStringRepresentation = Optional.of("(" +Arrays.stream(expressions).map(Object::toString).collect(Collectors.joining("&"))+")");
````
is apparently also available

Reply:
Yes, that is correct. You can also include the prefix and suffix in Collectors.joining call.